moveit_ros/hybrid_planning/test/config/local_planner.yaml

```diff
@@ -0,0 +1,14 @@
+
+robot_description: "robot_description"
+trajectory_operator_plugin_name: "moveit_hybrid_planning/SimpleSampler"
+local_constraint_solver_plugin_name: "moveit_hybrid_planning/ForwardTrajectory"
+local_planning_frequency: 100.0
+global_solution_topic: "global_trajectory"
+local_solution_topic: "/panda_joint_group_position_controller/commands" # or panda_arm_controller/joint_trajectory
+local_solution_topic_type: "std_msgs/Float64MultiArray" # or trajectory_msgs/JointTrajectory
+
+# SimpleSampler param
+pass_through: false
+
+# ForwardTrajectory param
+stop_before_collision: false
```

Comment:
I think this should be `true` so the demonstration runs like the provided gif.

If this is false, the demo robot passes through the objects when I try it